pkg/kubelet/kubelet.go

```diff
@@ -1444,6 +1455,15 @@ func (kl *Kubelet) GetPodByName(namespace, name string) (*api.Pod, bool) {
 	return kl.podManager.GetPodByName(namespace, name)
 }
 
+func (kl *Kubelet) updateRuntimeUp() {
+	err := waitUntilRuntimeIsUp(kl.containerRuntime, 0)
```

Comment:
I don't think we can use 0 as a value here. I think [the check](https://github.com/wojtek-t/kubernetes/blob/kubelet_removing_nodes/pkg/kubelet/kubelet.go#L103) will always fail (or be racy)